config/initializers/homescreen.rb

```diff
@@ -72,36 +72,31 @@
 end
 
 OpenProject::Static::Homescreen.manage :links do |links|
-  link_hash = OpenProject::Static::Links.links
-
   links.push(
     {
       label: :user_guides,
       icon: "icon-context icon-rename",
-      url: link_hash[:user_guides][:href]
+      url_key: :user_guides
     },
     {
       label: :glossary,
       icon: "icon-context icon-glossar",
-      url: link_hash[:glossary][:href]
+      url_key: :glossary
     },
     {
       label: :shortcuts,
       icon: "icon-context icon-shortcuts",
-      url: link_hash[:shortcuts][:href]
+      url_key: :shortcuts
     },
     {
       label: :forums,
       icon: "icon-context icon-forums",
-      url: link_hash[:forums][:href]
+      url_key: :forums
+    },
+    {
+      label: :impressum,
+      url_key: :impressum,
+      icon: "icon-context icon-info1"
```

Comment:
very minor, but the order of the arguments should be the same for all items

```suggestion
      label: :impressum,
      icon: "icon-context icon-info1",
      url_key: :impressum
```